.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!-- 
+Insert your contribution description here.
+Ensure your contribution fills the Contributing Guidelines described here:
+https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md 
+-->
+
+<!-- 
+Special Attention to TITLE format
+https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md#title
```

Comment:
it works as soon the new contributing file is available. e.g.: https://github.com/bonitasoft/bonita-doc/blob/7.3/README.md#taxonomy